src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -28,6 +28,11 @@ struct SpatialMetric : db::SimpleTag {
   static std::string name() noexcept { return "SpatialMetric"; }
 };
 template <size_t Dim, typename Frame, typename DataType>
+struct DetAndInverseSpatialMetric : db::SimpleTag {
```

Comment:
I would like to avoid this SimpleTag if possible, as I would rather just have the compute item create the individual `DetSpatialMetric and InverseSpatialMetric`.   I think this should be possible by having the compute item return a `Variables<DetSpatialMetric, InverseSpatialMetric>` as the individual items in a Variables are added . @nilsdeppe please comment  (This would then avoid the additional compute items in #1469 that copy the determinant and inverse from the pair.)

Reply:
I like this simplification. Perhaps, something like this should do:

```
template <size_t SpatialDim, typename Frame, typename DataType>
struct ModifiedDetAndInverseSpatialMetricCompute
    : ::Tags::Variables<
          tmpl::list<DetSpatialMetric<DataType>,
                     InverseSpatialMetric<SpatialDim, Frame, DataType>>>,
      db::ComputeTag {
  using argument_tags = tmpl::list<SpatialMetric<SpatialDim, Frame, DataType>>;
  using base = ::Tags::Variables<
      tmpl::list<DetSpatialMetric<DataType>,
                 InverseSpatialMetric<SpatialDim, Frame, DataType>>>;
  using type = typename base::type;
  static constexpr auto function(
      const tnsr::ii<DataType, SpatialDim, Frame>& spatial_metric) {
    const auto det_and_inv_pair = determinant_and_inverse(spatial_metric);
    tuples::TaggedTuple<DetSpatialMetric<DataType>,
                        InverseSpatialMetric<SpatialDim, Frame, DataType>>
        det_and_inv_tupl;
    get<DetSpatialMetric<DataType>>(det_and_inv_tupl) = det_and_inv_pair.first;
    get<InverseSpatialMetric<SpatialDim, Frame, DataType>>(det_and_inv_tupl) =
        det_and_inv_pair.second;
    const auto result = variables_from_tagged_tuple(det_and_inv_tupl);
    return result;
  };
};
```
?

Reply:
@kidder : I put the above compute item in with aa760df443774332cfd50bc407c56514356079d6 as a temporary tag with "Modified" in its name. It has passed through Travis tests. Since finalizing this change requires changes to this + another PR, I was hoping for an approval of it beforehand. So, do let me know when you're satisfied with the above code.

Reply:
I'm happy with removing the one you suggest removing :) I'm in agreement with @kidder on the Variables being the right interface :)

Reply:
Also I would just change the function `determinant_and_inverse` to return the `Variables`.  Then you don't need to know which one is `.first` or `.second`.  But yes, this is what I had in mind.

Reply:
done